pages/common/wezterm.md

```diff
@@ -0,0 +1,37 @@
+# wezterm
+
+> Wez's Terminal Emulator, a powerful cross-platform terminal emulator and multiplexer written by wez and implemented in Rust.
+> Usage: wezterm [OPTIONS] [COMMAND].
+> More information: <https://wezterm.org>.
```

Comment:
```suggestion
> Wez's Terminal Emulator, a powerful cross-platform terminal emulator and multiplexer written by wez and implemented in Rust.
> More information: <https://wezterm.org>.
```
Not needed. Check other pages and the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#pages) for reference on how the pages are structured